src/ServiceControl.Audit/Infrastructure/WebApi/WebApiHostBuilderExtensions.cs

```diff
@@ -32,14 +42,14 @@ public static IHostBuilder UseWebApi(this IHostBuilder hostBuilder, string rootU
             return hostBuilder;
         }
 
-        static void RegisterInternalWebApiControllers(ContainerBuilder containerBuilder)
+        static void RegisterInternalWebApiControllers(IServiceCollection serviceCollection)
         {
             var controllerTypes = Assembly.GetExecutingAssembly().DefinedTypes
                 .Where(t => typeof(IHttpController).IsAssignableFrom(t) && t.Name.EndsWith("Controller", StringComparison.Ordinal));
 
             foreach (var controllerType in controllerTypes)
             {
-                containerBuilder.RegisterType(controllerType);
+                serviceCollection.AddScoped(controllerType);
```

Comment:
added them as scoped for now but previously they have been transient from what I can tell.